app/views/docs/upgrade.phtml

```diff
@@ -3,35 +3,66 @@
+    <p>You do not need to run migration when upgrading to a minor version. For example, going from 0.13.0 to 0.13.2 would not require running migration.</p>
+
+    <p>It is highly recommended to <a href="https://gist.github.com/Meldiron/47b5851663668102a676aff43c6341f7" target="_blank" rel="noopener">backup your server</a> data before running the migration. It is recommended to run the migration process on a dev instance and make sure your application is working well and that you have checked for any breaking changes in the new version <a href="https://github.com/appwrite/appwrite/tags" target="_blank" rel="noopener">changelog.</a></p>
 </div>
 
 <!-- <p>When updating to a new Appwrite version, make sure to run the <a href="/docs/installation">installation script for the new version</a> from the same location, you executed your original setup. This will allow the installation script to update your existing docker-compose.yml file. The correct location for execution is the parent directory of your existing appwrite directory.</p> -->
 
 <!-- <p>The data migration tool will allow you to easily migrate your current Appwrite data to work with the new version. </p> -->
 
-<p> The first step is to install the latest version of Appwrite. Head to the directory where you ran your previous Appwrite install command. 
+<p>The first step is to install the latest version of Appwrite. Head to the directory where you ran your previous Appwrite install command. </p>
 
-<div class="ide margin-bottom" data-lang="bash" data-lang-label="Bash">
-    <pre class="line-numbers"><code class="prism language-bash" data-prism>
-    parent_directory  <= you run the command in this directory
+<div class="ide margin-bottom">
+    <pre class="line-numbers"><code class="prism" data-prism>parent_directory <= you run the command in this directory
     └── appwrite
-        └── docker-compose.yml
-    </code></pre>
+        └── docker-compose.yml</code></pre>
 </div>
 
-This is the parent directory, where you will find the <b>appwrite</b> directory that contains the <b>docker-compose.yml</b> and <b>.env</b> files.</p>
+<p>This is the parent directory where you will find the <b>appwrite</b> directory, inside which there are <b>docker-compose.yml</b> and <b>.env</b> files.</p>
+
+<h3><a href="/docs/installation#unix" id="unix">Unix</a></h3>
 
 <div class="ide margin-bottom" data-lang="bash" data-lang-label="Bash">
     <pre class="line-numbers"><code class="prism language-bash" data-prism>docker run -it --rm \
     --volume /var/run/docker.sock:/var/run/docker.sock \
     --volume "$(pwd)"/appwrite:/usr/src/code/appwrite:rw \
     --entrypoint="install" \
-    appwrite/appwrite:<?php echo APP_VERSION_STABLE; ?></code></pre>
+    appwrite/appwrite:[NEXT MAJOR VERSION]</code></pre>
```

Comment:
Why did we want to do this ? 

`APP_VERSION_STABLE` already contains this value and give developers a direct copy paste code snippet

Reply:
I did this because people frequently ran this code and it would take them from 0.11 to 0.13, for example... Which breaks their migration.

We wanted to do this. What's your opinion on this?

Reply:
Let's use the latest version in the docs so as to provide an easy copy and paste snippet and let's handle the check in the installation script

Reply:
@christyjacob4 Yep let me push my changes up and we'll be good